src/compounds/ad-banner/package.json

```diff
@@ -12,6 +12,7 @@
     "react": "^16.7.0"
   },
   "dependencies": {
+    "@uswitch/trustyle.awards-tag": "^0.1.1",
     "@uswitch/trustyle.badge": "^2.0.2",
```

Comment:
if this is no longer being used it would probably be best to remove it from the package.json